frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.tsx

```diff
@@ -143,21 +133,28 @@ const DataSelectorTablePicker = ({
           {isSavedQuestionList && (
             <LinkToDocsOnReferencingSavedQuestionsInQueries />
           )}
-        </Container>
+        </Box>
       </DelayGroup>
     );
   } else {
     return (
       <Section>
         <DataSelectorSectionHeader header={header} />
-        <NoTablesFound>{t`No tables found in this database.`}</NoTablesFound>
+        <Box p="xl" ta="center">{t`No tables found in this database.`}</Box>
```

Comment:
Before (64px)
![image](https://github.com/metabase/metabase/assets/31325167/7974cb54-46f9-4cca-af8c-91bd8c1b10e6)

After (32px)
![image](https://github.com/metabase/metabase/assets/31325167/a68452de-6aef-4ed0-a3dc-d1ca28812dcb)

If @metabase/design feels strongly about using the larger padding here, I can add it as the `style` prop.

Reply:
@nemanjaglumac you can always set `p="4rem"`